### PR TITLE
Lei/bag parser

### PR DESCRIPTION
**Pull request by liulei1984**
training data generation pipe line
@ByteShaker @miharothl @tho15 @sndre

### CODE REVIEW

File: ros/src/bag_parser/scripts/parse_bag.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+bridge = CvBridge()
+
+def conv2car_coord(car_pose, g_position):
+	l_position = g_position
+	
+	pos = car_pose.position
+	qor = car_pose.orientation
+	quaternion = (qor.x, qor.y, qor.z, qor.w)
+	euler = tf.transformations.euler_from_quaternion(quaternion)
```
Comment (sndre) [inline ros/src/bag_parser/scripts/parse_bag.py:23]: I believe you can just pass `qor` without creating intermediate `quaternion `

File: ros/src/bag_parser/scripts/parse_bag_lei.py [hunk 2]
```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python
```
Comment (sndre) [inline ros/src/bag_parser/scripts/parse_bag_lei.py:1]: why do we have two different `parse_bag` scripts?

**Review by sndre — COMMENTED** [review]
Just a couple questions below 👇🏻

File: ros/src/bag_parser/scripts/parse_bag_lei.py (hunk 2 above)
Comment (liulei1984) [reply ros/src/bag_parser/scripts/parse_bag_lei.py:1]: parse_bag.py is thomas's first data generation pipeline implementation
I cloned thomas's branch and improved his pipeline which is pars_bag_lei.py

Comment (sndre) [reply ros/src/bag_parser/scripts/parse_bag_lei.py:1]: @liulei1984, do we have to commit both of these? seems like one is improved copy of the other.

**Pull request merged**